src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -181,6 +181,15 @@ private bool UsePlaceholderChannel(IRpcWorkerChannel channel)
                     return false;
                 }
 
+                // We support specialization of dotnet-isolated only on 64bit host process.
+                var is64Bit = _environment.Is64BitProcess;
+                _logger.LogDebug("Is64BitProcess: {is64Bit}", is64Bit.ToString());
+
+                if (!is64Bit)
+                {
```

Comment:
Or rather add the log here when we skip placeholders based on this logic?

Reply:
Changed the log statement and location.